README.md

```diff
@@ -124,29 +127,47 @@ class MyComponent {
 }
 ```
 
-The HTTP will include the following Connection types, and will provide an interface to ensure third-
-party Connection implementations conform to the expected API.
+The Http library will include the following `Connection` factories, and will provide an interface to
+ensure third-party `Connection` implementations conform to the expected API.
 
- * XHRConnection
- * JSONPConnection
- * NodeConnection
- * MockConnection
+ * Http
+ * HttpJsonp
+ * HttpNode
+ * HttpMock
+ * _HttpFetch_ TBD
 
 ### Connection
 
-Connections are constructed with a single argument, a `ConnectionConfig` object.
+Connections are constructed with a single argument, either a url string, or a `ConnectionConfig`
+object.
 
-Using a callable function "create" instead of instantiating connections via "new" keyword enables
-creating connections in part of a functional chain without having to wrap the instantiation in a
-closure. A `Connection` implementation may be instantiable with the "new" keyword.
+Using a callable function that accepts a single argument instead of instantiating connections via
+"new" keyword enables creating connections in part of a functional chain without having to wrap the
+instantiation in a closure. The requirement of using a factory function is only imposed on the
+officially-supported `Connection` factories. Other conforming implementations may instantiate
+Connections in whatever way suits the Connection, but the returned value should conform to the
+`Observable` interface.
```

Comment:
it would be good if these interface names were replaced with links to wherever these are defined, if such a place exists